dask/compatibility.py

```diff
@@ -1,24 +1,10 @@
-import importlib.metadata
 import sys
 
+from importlib_metadata import entry_points
 from packaging.version import parse as parse_version
 
+__all__ = ["entry_points"]
+
```

Comment:
If `importlib_metadata.entry_points` removes the need for the existing `dask.compatibility.entry_points` utility then we can just remove this `import` here and replace `dask.compatibility.entry_points` usages with `importlib_metadata.entry_points` instead 

cc @douglasdavis for visibility (I don't think this utility is used in other projects -- feel free to correct me if that's not the case)

Reply:
Yep sounds good

Reply:
There shouldn't be any use of that function outside of dask/dask!